File: react-ui/src/components/Car.js
import React from'react'

const Car = (props) => {
    return (
        <div>
            <h3>Price: {props.price}</h3>
            <h4>Make: {props.make}</h4>
            <h4>Model: {props.model}</h4>
            <h4>Miles: {props.miles}</h4>
            <h4>Description: {props.description}</h4>
            <h4>Owner: {props.owner}</h4>
            <hr />
        </div>
    );
};

export default Car

File: cars-api/src/main/java/com/example/carsapi/Models/Car.java
package com.example.carsapi.Models;

import lombok.*;

import javax.persistence.*;
import java.util.ArrayList;

@Data
@AllArgsConstructor @NoArgsConstructor @Getter @Setter
@Entity @Table(name = "cars")
public class Car {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "make")
    private String make;

    @Column(name = "model")
    private String model;

    @Column(name = "miles")
    private int miles;

    @Column(name = "picture_urls")
    private ArrayList<String> pictures;

    @Column(name = "owner")
    private String owner;

}